### PR TITLE
add debug messages

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

File: flit/wheel.py [hunk 1]
```diff
@@ -45,11 +47,21 @@ def clean_build_dir(self):
...
+
+
         mod =  self.module
         # Copy module/package to build directory
         if mod.is_package:
             ignore = shutil.ignore_patterns('*.pyc', '__pycache__')
-            shutil.copytree(str(mod.path), str(self.build_dir / mod.name), ignore=ignore)
+            shutil.copytree(str(mod.path), str(self.build_dir / mod.name),
+                    ignore=ignore,
+                    symlinks=True,
```
Comment (takluyver) [inline flit/wheel.py]: Is there any use in setting this? It's either going to fail later or lose the symlink relationships when it creates the zip file, right?

Comment (Carreau) [inline flit/wheel.py]: Slipped through `add -p`.

**Comment by Carreau** [comment]
fixed.

File: flit/wheel.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 from . import common
 from . import inifile
 
+from shutil import copy2, copytree
```
Comment (takluyver) [inline flit/wheel.py]: shutil is already imported above - for neatness, let's use `shutil.copy2` below and drop the extra import

**Comment by Carreau** [comment]
done.

**Pull request merged**